lib/processors/manifestCreator.js

```diff
@@ -667,7 +672,8 @@ module.exports = function({libraryResource, resources, options}) {
 		return Promise.resolve(null); // a fulfillment of null indicates that no manifest has been created
 	}
 
-	return createManifest(libraryResource, libBundle, options.descriptorVersion, options.include3rdParty)
+	return createManifest(libraryResource, libBundle, options.descriptorVersion, options.include3rdParty,
+		options.minUI5VersionRequired)
```

Comment:
I would prefer a switch to simply omit the minUI5Version instead of _trying_ to resolve it. Maybe `omitMinUI5Version`?

In the ui5-server use case, there is no benefit from having the minUI5Version available in _some_ cases. So I think it should never be available in order to keep things simpler.

Reply:
As discussed, let's call it omitMinVersions because not only the minUI5Version but also the minVersion of dependencies could lead to issues. The added versionInfoGenerator test proves that missing (empty string) minVersion/minUI5Version info doesn't affect the result of a sap-ui-version.json